src/mixxx.cpp

```diff
@@ -843,11 +851,40 @@ void setVisibilityOptionState(QAction* pAction, ConfigKey key) {
     pAction->setChecked(pVisibilityControl != NULL ? pVisibilityControl->get() > 0.0 : false);
 }
 
-void MixxxMainWindow::toggleCheckedSamplers() {
-    m_pViewShowSamplers->blockSignals(true);
+void MixxxMainWindow::toggleCheckedMenuAction(QAction* menuAction, ConfigKey key) {
```

Comment:
I do associate the term toogle with inverting the value. IMHO updateCheckedMenuAction fits better.